src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,80 @@
+package duke;
+
+import duke.command.AddCommand;
+import duke.command.CheckCommand;
+import duke.command.Command;
+import duke.command.DeleteCommand;
+import duke.command.DoneCommand;
+import duke.command.ExitCommand;
+import duke.command.FindCommand;
+import duke.command.ListCommand;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+/**
+ * The <code>Parser</code> reads a command from user and perform various checks to determine the next action.
+ */
+public class Parser {
+    private static final String ignoreCase = "(?i)";
+    private static final String wildcard = "(.*)";
```

Comment:
Perhaps these variable names could follow the format for naming constants?

Reply:
Seems like I overlooked that... Thanks for the review!